### ISSUE TITLE
Proof Points

### ISSUE THREAD

**Issue by c-schwarze** [betheluniversity/tinker#302]
This is a low priority at the moment. Before starting with this project, talk with Mike Vedders.

Marketing would like to have a way to manage the history of proof points. This may or may not become a tinker module. 

Key features
* add a link field for proof points (this should be grouped with the "data" in some way)
* maintain the history of proof points
* be able to expire or "rotate" those info sections.
     * have a way to highlight proof points that are out of date.

**Comment by c-schwarze** [comment]
We talked about this in our cascade meeting.

We have decided to use the current features. We are using cascade's archive and cascade's version history to view old proof points
